lms/static/sass/discussion/utilities/_shame.scss

```diff
@@ -29,6 +29,19 @@
   position: relative;
 }
 
+// Temporary breadcrumbs
+.has-breadcrumbs {
```

Comment:
Updated to use as much UXPL styling as possible. Some margin fixes are still necessary so this selector remains.